docs/modules/ROOT/partials/getting-started/proc-setting-up-kafka-streams-storage.adoc

```diff
@@ -75,16 +75,38 @@ spec:
     streams:
       bootstrapServers: "my-cluster-kafka-bootstrap.my-project.svc:9092"
 ----
-      
+
 . Click *Create* and wait for the {registry} route to be created on OpenShift.
 
 . Click *Networking* > *Route* to access the new route for the {registry} web console. For example:
 +
 [source]
 ----
-http://example-apicurioregistry.my-project.my-domain-name.com/   
+http://example-apicurioregistry.my-project.my-domain-name.com/
 ----
 
+== Overriding default Kafka topic names
+You can change the default names of the Kafka topics that {registry} will use to store data in Kafka.  It is sometimes
+necessary to do this when sharing the Kafka cluster with other applications that may already be using topics named
+`storage-topic` or `global-id-topic`.
+
+Change the default topic names by overriding them either by setting appropriate environment variables *or* by
+setting appropriate Java system properties:
```

Comment:
Suggest showing a quick examples of setting by env var and Java system properties?